python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -323,6 +324,16 @@ def __call__(self, data: KeyedData, **kwargs) -> Vector:
         return cast(Vector, data[key])
 
 
+class PsfStarSelector(FlagSelector):
+    bands = ListField[str](
+        doc="The bands to apply the flags in, takes precedence if band supplied in kwargs",
+        default=["i"],
+    )
```

Comment:
Is this what you want the default behaviour to be? If you want it to default to the band the plot is being made in then set this to [].